File: Program38.cpp
#include<iostream>
using namespace std;

class Number
{
    private:
      int iNo;

    public:
      void Accept()
      {
          cout<<"Enter number"<<endl;
          cin>>this->iNo;
      }
      void DisplayF()
      {
          int iCnt = 0;
          cout<<"Forward Display"<<endl;
          for(iCnt = 1; iCnt <= iNo; iCnt++)
          {
              cout<<iCnt<<"\t";
          }
      }
      void DisplayB()
      {
          int iCnt = 0;
           cout<<"Backward Display"<<endl;

          for(iCnt = iNo; iCnt >= 1; iCnt--)
          {
              cout<<iCnt<<"\t";
          }
      }
};

int main()
{
    Number nobj;

    nobj.Accept();
    nobj.DisplayF();
    nobj.DisplayB();

    return 0;
}

File: Program38.java
import java.lang.*;
import java.util.*;

class Number
{
    private int iNo;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number");
        this.iNo = sobj.nextInt();
    }
    public void DisplayF()
    {
        int iCnt = 0;
        System.out.println("Forward Display");

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.println(iCnt);
        }
    }
    public void DisplayB()
    {
        int iCnt = 0;
        System.out.println("Backward Display");

        for(iCnt = iNo; iCnt >= 1; iCnt--)
        {
            System.out.println(iCnt);
        }
    }
}
class Program38
{
    public static void main(String arg[])
    {
        Number nobj = new Number();

        nobj.Accept();
        nobj.DisplayF();
        nobj.DisplayB();
    }
}